src/mp_game_settings.hpp

```diff
@@ -85,8 +85,9 @@ struct mp_game_settings
 	{
 		boost::optional<version_info> version;
 		boost::optional<version_info> min_version;
-
 		std::string name;
+		bool required;
+		config content;
```

Comment:
It be better to store this into a `struct addon_content { string type: string id; }; std::vector<addon_content> content;` instead ofa  config object;

Reply:
Updated.